src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -80,6 +80,19 @@ public async Task<MsalTokenResponse> AcquireTokenUsingBrokerAsync(Dictionary<str
 
         private async Task AcquireTokenInternalAsync(IDictionary<string, string> brokerPayload)
         {
+            if (brokerPayload.ContainsKey(BrokerParameter.BrokerInstallUrl))
+            {
+                _logger.Info("Android Broker - broker payload contains install url");
+
+                var appLink = AndroidBrokerHelper.GetValueFromBrokerPayload(brokerPayload, BrokerParameter.BrokerInstallUrl);
```

Comment:
nit: not a big fan of this helper, a null is easier to track than an empty string when debugging.